composer.json

```diff
@@ -15,7 +15,6 @@
     ],
     "require": {
         "composer-plugin-api": "^1.1",
-        "composer/composer": "^1.8@stable",
```

Comment:
The version up to 1.8 may be required for some reason. Can you use the double pipe to support 1.8+ OR 2.x https://getcomposer.org/doc/articles/versions.md#version-range